src/api/messages.js

```diff
@@ -0,0 +1,58 @@
+import request from './request'
+import { struct } from 'superstruct'
+
+const Message = struct({
+  text: 'string',
+  sendAt: 'date',
+  userId: 'number',
+  User: 'Object',
+  taskId: 'number',
+  task: 'Object',
+})
+
+export default {
+  /**
+   * Get message and if id is undefined you get all messages
+   * @param {Number} id
+   * @return {Promise}
+   */
+  async get(id) {
+    const res = await request.get(id ? `messages/${id}` : 'messages')
+    return await res.json()
+  },
+
+  /**
+   * Create message
+   * @param {Object} message
+   * @return {Promise}
+   */
+  async create(message) {
+    //Validate data for the body
+    Message(message);
+    const path = 'messages?password=' + message.password
```

Comment:
This is not valid for messages

Reply:
nej det kan jeg sku' godt se det gik lige lidt for hurtigt der